src/bucket/test/BucketListTests.cpp

```diff
@@ -645,38 +643,30 @@ TEST_CASE("BucketList number dump", "[bucket][bucketlist][count][!hide]")
 {
     for (uint32_t level = 0; level < BucketList::kNumLevels; ++level)
     {
-        CLOG(INFO, "Bucket")
-            << "levelSize(" << level
-            << ") = " << formatU32(BucketList::levelSize(level))
-            << " (formally)";
+        CLOG_INFO(Bucket, "levelSize({}) = {} (formally)", level,
```

Comment:
looks like `formatU32` is now obsolete, can just be done with the format string?

Reply:
it's a helper that prints the value in two forms (decimal and hex).